app/src/main/res/layout/item_public_restaurant.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The file name isn't obvious on what this layout is supposed to really represent... Also, I suggest a generic reusable bullet list layout and epoxy model so it can be reused in many screens